rtsf-at-checkout-event-reconciler/events/process_events.go

```diff
@@ -9,12 +9,18 @@ import (
 	"math"
 	"time"
 
-	"github.com/edgexfoundry/app-functions-sdk-go/appcontext"
-	"github.com/edgexfoundry/go-mod-core-contracts/models"
+	"github.com/edgexfoundry/app-functions-sdk-go/v2/pkg/interfaces"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/clients/logger"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/models"
 
+	"event-reconciler/config"
 	"event-reconciler/rfidgtin"
 )
 
+type EventsProcessor struct {
+	ProcessConfig *config.ReconcilerConfig
```

Comment:
Add global vars from below to here and add a factory method that initializes the ones being set from main using the passed in config. All should be private vars. If access needed create Getter/Setter functions.